services/backend/src/service/project_strategy.js

```diff
@@ -42,7 +44,7 @@ module.exports = (strategyPersistence) => ({
       error.code = 400;
       throw error;
     }
-
-    return strategyPersistence.findSelectedStrategiesGeoJson(pId);
+    const selectedStrategies = await strategyPersistence.findSelectedStrategiesGeoJson(pId);
+    return uploadFile(selectedStrategies);
```

Comment:
I added a try/catch statement so that the error message returned is changed for a less explicit one and the real error is logged. Please let me know your opinions.

![imagen](https://user-images.githubusercontent.com/19979979/156473634-5380d547-d618-4b6d-884d-af5b44b2fc5c.png)